Khan/perseus
# Implement adjusting colors for locked points

## Summary:
Now that the interactive graph editor can add locked
points, we also need to make it so that they can update
the color of those points.

In this PR, I added a dropdown that allows for changing
the color. I set the default color to blue. I also
made is so that the actual color value isn't saved
as part of the locked figure styles, just the name.
This then gets converted to the wonder blocks color
later in the mafs graph. I did it this way to avoid
having to convert back and forth between the color name
and value in the color UI.

Issue: https://khanacademy.atlassian.net/browse/LEMS-1751

## Test plan:
`yarn jest packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor.test.tsx`
`yarn jest packages/perseus/src/widgets/__tests__/interactive-graph.test.ts`

Storybook
- Go to http://localhost:6006/?path=/story/perseus-widgets-interactive-graph--segment-with-mafs-and-locked-points
- Confirm that the locked points are blue by default
- Go to http://localhost:6006/?path=/story/perseus-editor-widgets-interactive-graph-editor--with-locked-points
- There should be two locked points: one blue and one gold
- Confirm that using the color dropdown changes the color for the appropriate point
- Add a new point
- Confirm that it's blue by default
- Change its color and confirm all the colors work
- Change the coordinates of the points and make sure that nothing breaks
- Delete points and make sure nothing breaks
- Add points again and make sure nothing breaks 
- Click on the "Color" label and confirm that it opens the color dropdown.
- ^ Repeat for each dropdown and make sure it opens the correct one (IDs are unique and work as expected)

## Demo:

<img width="681" alt="Screenshot 2024-03-14 at 3 19 08 PM" src="https://github.com/Khan/perseus/assets/13231763/95fe6224-0d16-40bd-aa05-3982b2c9a40c">

https://github.com/Khan/perseus/assets/13231763/6c8306b1-8364-4096-bfb7-4994db3a33e7
